### ISSUE TITLE
Check "IP Location Block" based on "IP Geo Block"

### ISSUE THREAD

**Issue by gdarko** [tokkonopapa/WordPress-IP-Geo-Block#65]
I forked the project and fixed all the problems that the current outdated version had.

https://wordpress.org/plugins/ip-location-block/
https://github.com/gdarko/ip-location-block/

### Some notable changes
- Updated code to match the latest geolocation APIs
- Implemented key for Maxmind database (maxmind requires key as of 2018)
- Replaced Google Maps with Leaflet / OSM
- Fixed various warnings and made it compatible with most recent PHP
- Added migration tool
- much more...

### Migration
1. Deactivate **IP Geo Block** but don't remove it _yet_ because it will remove the settings as well.
2. In IP Location Block, use the "**Migrate from IP Geo Block**" option at the bottom to copy the settings from **IP Geo Block**. 
3. You can now remove **IP Geo Block**

**Comment by darrenmcentee** [comment]
Brilliant Darko, really appreciate the effort on this. Look forward to the updates and advancements.
Did you manage to make contact with tokkonopapa?

**Comment by gdarko** [comment]
Hi @darrenmcentee , I tried to contact tokkonopapa via email about two months ago, but there is no response.

**Comment by darrenmcentee** [comment]
Yeah, I also tried to make contact with tokkonopapa, probably over a year ago, with no response either. 
One quick question on your fork, will the json settings from the original version still import as before, or should the settings be manually set and json re-exported as a new file? Thanks.

**Comment by gdarko** [comment]
Hi @darrenmcentee ,

The option for Migration is now available in the "Plugin Settings" section as of version 1.0.3

You just need to click the button and it will import all the settings. Logs will not be migrated. 

Also, make sure you verify the provider's list after migration because only IP2Location, IP-API, and GeoIPDB work nowadays without a key.

Best Regards,
Darko
